pyproject.toml

```diff
@@ -7,19 +7,32 @@ name = "advent-of-code-data"
 version = "2.0.1"
 description = "Get your puzzle data with a single import"
 requires-python = ">=3.9"
-classifiers = [
+classifiers = [  # https://pypi.org/classifiers/
+    "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
-    "Topic :: Software Development :: Libraries",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
     "Topic :: Games/Entertainment :: Puzzle Games",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Typing :: Typed",
```

Comment:
Apart from `"Typing :: Typed"`, which should be added, please leave the rest of these unchanged. They're annoying to maintain when new Python versions are released or old ones are EOL, and the classifiers are of questionable utility anyway.